kinetics_MO.py

```diff
@@ -0,0 +1,261 @@
+import numpy as np
+import pandas as pd
+import pyomo.environ as pyo
+from measure_optimize import (
+    MeasurementOptimizer,
+    SensitivityData,
+    MeasurementData,
+    CovarianceStructure,
+    ObjectiveLib,
+)
+import pickle
+import time
+
+
+# set up problem formulation
+
+# number of time points for DCM
+Nt = 8
+
+# maximum manual measurement number for each measurement
+max_manual_num = 10
+# minimal measurement interval
+min_interval_num = 10.0
+# maximum manual measurement number for all measurements
+total_max_manual_num = 10
+
+# index of columns of SCM and DCM in Q
+static_ind = [0, 1, 2]
+dynamic_ind = [3, 4, 5]
+# this index is the number of SCM + nubmer of DCM, not number of DCM timepoints
+all_ind = static_ind + dynamic_ind
+num_total_measure = len(all_ind)
+
+# meausrement names
```

Comment:
Are these the columns in the CSV file? If so, declare these first and then declare the indices.

Reply:
- [ ] add an example of csv file header in this file

Reply:
added headers in both files